src/main/java/Response/Response.java

```diff
@@ -31,6 +37,17 @@ public Builder reasonPhrase(String reasonPhrase) {
       this.reasonPhrase = reasonPhrase;
       return this;
     }
+
+    public Builder contentType(String contentType) {
+      this.headers.put(ResponseHeader.CONTENT_TYPE, contentType);
+      return this;
+    }
+
+    public Builder contentLength(int contentLength) {
+      String stringifedContentLength = Integer.toString(contentLength);
+      this.headers.put(ResponseHeader.CONTENT_LENGTH, stringifedContentLength);
+      return this;
+    }
```

Comment:
I wonder what the pros/cons are to using a setHeader method` for the builder instead of implementing specific builder methods for each header (and then the handler would build up the response calling setHeader however many times to add to the header map as it builds the response). I think you mentioned that your current implementation means you need to add different methods for each header you support in this class. The more generic `setHeader` idea means that the handlers are responsible for adding the headers that they need.

Reply:
I like the idea of one `setHeader` method. My `Response` class doesn't have to grow with every new possible header. It doesn't even know what headers are possible. I've created and tested the `setHeader(String headerField, int value)` and `setHeader(String headerField, String value)`. If it's OK, I'd like to make that change either in its own separate branch or after I've merged `ft/image-content-negotation` to master. That branch made changes to `Response` and each Handler in order populate `messageBody` with `byte[]` rather than `String` (making it possible to respond with images). Otherwise, I think that my git conflicts are going to get super messy.

Reply:
ok